config.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+function pre_build {
+    set -ex
+    build_zlib
+    pip install cmake
+    pip install .
+    pip install pytest
+    pytest .
```

Comment:
why are the tests run in the pre_build rather than run_tests?

Reply:
in run_tests installs package from a wheel and does not have source code (in linux builds that happens in a separate container that only mounts folder with wheel)... I suspect it is possible to locate source folder in which pytest should run tests, but it would be rather hacky.